doc/move_group_interface/launch/move_group.launch.py

```diff
@@ -0,0 +1,118 @@
+    ompl_planning_pipeline_config['move_group'].update(ompl_planning_yaml)
+
+    # Trajectory Execution Functionality
+    controllers_yaml = load_yaml('moveit2_tutorials', 'config/controllers.yaml')
+    moveit_controllers = { 'moveit_simple_controller_manager' : controllers_yaml,
+                           'moveit_controller_manager': 'moveit_simple_controller_manager/MoveItSimpleControllerManager'}
+
+    trajectory_execution = {'moveit_manage_controllers': True,
+                            'trajectory_execution.allowed_execution_duration_scaling': 1.2,
+                            'trajectory_execution.allowed_goal_duration_margin': 0.5,
+                            'trajectory_execution.allowed_start_tolerance': 0.01}
+
+    planning_scene_monitor_parameters = {"publish_planning_scene": True,
+                 "publish_geometry_updates": True,
+                 "publish_state_updates": True,
+                 "publish_transforms_updates": True}
+
+    # Start the actual move_group node/action server
+    move_group_node = Node(package='moveit_ros_move_group',
+                           executable='move_group',
+                           output='screen',
+                           parameters=[robot_description,
+                                       robot_description_semantic,
+                                       kinematics_yaml,
+                                       ompl_planning_pipeline_config,
+                                       trajectory_execution,
+                                       moveit_controllers,
+                                       planning_scene_monitor_parameters])
+
+    # RViz
+    rviz_config_file = get_package_share_directory('moveit2_tutorials') + "/launch/move_group.rviz"
+    rviz_node = Node(package='rviz2',
+                     executable='rviz2',
+                     name='rviz2',
+                     output='log',
+                     arguments=['-d', rviz_config_file],
+                     parameters=[robot_description,
+                                 robot_description_semantic,
+                                 ompl_planning_pipeline_config,
+                                 kinematics_yaml])
+
+    # Static TF
+    static_tf = Node(package='tf2_ros',
+                     executable='static_transform_publisher',
+                     name='static_transform_publisher',
+                     output='log',
+                     arguments=['0.0', '0.0', '0.0', '0.0', '0.0', '0.0', 'world', 'panda_link0'])
+
+    # Publish TF
+    robot_state_publisher = Node(package='robot_state_publisher',
+                                 executable='robot_state_publisher',
+                                 name='robot_state_publisher',
+                                 output='both',
+                                 parameters=[robot_description])
+
+    # Fake joint driver
+    fake_joint_driver_node = Node(package='fake_joint_driver',
```

Comment:
We replaced this node with the one from ros2_control [see](https://github.com/ros-planning/moveit2/blob/main/moveit_demo_nodes/run_move_group/launch/run_move_group.launch.py#L144)